src/v2/ion/uiSchemaTransformer.js

```diff
@@ -144,6 +144,44 @@ const createUISchema = (transformedResp, remediationForm) => {
     .flatten()
     .value();
 
+  const getCheckboxUiSchema = ({ label, type, required }) => ({
```

Comment:
those have been moved out this createUISchema function hence same for those 2 new functions you added.